### PR TITLE
Add eth spent over time metric

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: src/api/queries/blockchainQueries.js [hunk 1]
```diff
@@ -175,3 +175,19 @@ ApiClient_.prototype.fetchHistoricalBalance = function (slug, from, to, address)
...
+
+ApiClient_.prototype.fetchEthSpentOverTime = function (slug, from, to, limit, transactionType) {
+  var query = {
+    'query': '{ projectBySlug(slug: "' + slug + '"){\
+        ethSpentOverTime(from: "' + toUTC_(from) + '",\
+                         to: "' + toUTC_(to) + '",\
+                         interval: "1d"){\
+                               datetime\
+                               ethSpent\
+                             }\
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: Indentation is wrong here

File: test/functions/blockchainFunctionsTest.js [hunk 2]
```diff
@@ -57,3 +57,75 @@ describe('historicalBalance', () => {
...
+        }
+        ]
+      }
+    )
+
+    const response = san.ethSpentOverTime_(slug, from, to)
+
+    expect(response).to.deep.eq(
+      [
+        [
```
Comment (kalkov) [inline test/functions/blockchainFunctionsTest.js]: This also looks strange, can have a row per array, elements are just two.

Comment (IvanIvanoff) [reply test/functions/blockchainFunctionsTest.js]: Don't we run a formatter on this code? I think we could use prettier, for example?

File: src/api/queries/blockchainQueries.js [hunk 3]
```diff
@@ -226,3 +226,19 @@ ApiClient_.prototype.fetchEthTopTransactions = function (slug, from, to, limit,
 
   return this.conn.graphQLQuery(query, 'projectBySlug')
 }
+
+ApiClient_.prototype.fetchEthSpentOverTime = function (slug, from, to, limit, transactionType) {
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: Probably these arguments are leftovers? They are not used anywhere.

File: test/functions/blockchainFunctionsTest.js [hunk 4]
```diff
@@ -155,99 +154,74 @@ describe('tokenTopTransactions_', () => {
   })
 })
 
-describe('ethTopTransactions_', () => {
```
Comment (kalkov) [inline test/functions/blockchainFunctionsTest.js]: You need rebase master.

File: src/api/queries/blockchainQueries.js (hunk 3 above)
Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: Yes, you're right, sorry about that.

Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: Should be okay now.

File: test/functions/blockchainFunctionsTest.js (hunk 4 above)
Comment (spiderjako) [reply test/functions/blockchainFunctionsTest.js]: Yes, I saw, that it was a bit strange, it is fixed now.

**Pull request merged**